features/kinderen.feature

```diff
@@ -31,44 +31,48 @@ Functionaliteit: Kinderen van een persoon raadplegen
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 999996150                       |
+      | burgerservicenummer | 000001234                       |
       | fields              | kinderen.naam                   |
-      Dan heeft de response een persoon met een 'kind' met de volgende 'naam' gegevens
-      | naam          | waarde  |
-      | voornamen     | William |
-      | geslachtsnaam | Postma  |
-      En heeft de persoon een 'kind' met de volgende 'naam' gegevens
-      | naam          | waarde     |
-      | voornamen     | Sebastiaan |
-      | voorvoegsel   | de         |
-      | geslachtsnaam | Boer       |
-      En heeft de persoon een 'kind' met de volgende 'naam' gegevens
-      | naam          | waarde |
-      | voornamen     | Walter |
-      | voorvoegsel   | de     |
-      | geslachtsnaam | Boer   |
+      Dan heeft de response een persoon met een 'kind' met de volgende gegevens
+      | naam               | waarde  |
+      | naam.voornamen     | William |
+      | naam.geslachtsnaam | Postma  |
+	  | naam.voorletters   | W.      |
```

Comment:
Dit is een \@gba scenario. moet er dan wel een in de proxy afgeleide uitkomst beschreven worden?
Daarnaast staat het teveel naar rechts geschoven

Reply:
Is het niet de bedoeling dat alle scenario's end to end scenario's worden? De GBA API en de proxy worden toch uiteindelijk één?

Reply:
Is gedaan.